src/models/user.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
```

Comment:
Do you really have to use `'use strict'÷`? It's okay but it might have unforeseen effects. Just write your API without it. Unless if the setup you have requires it which I doubt.